techniques-et-culture.csl

```diff
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="sort-only" default-locale="fr-FR" page-range-format="expanded">
+  <info>
+    <title>Techniques &amp; culture (French)</title>
```

Comment:
@NMonteix, it looks like the title doesn't include spaces? See e.g. https://journals.openedition.org/tc/1553?lang=fr, which shows the journal referred to as "Techniques&Culture".

Reply:
It seems that it depends, in the end of the online papers the title is written with spaces (see e.g. [here](https://journals.openedition.org/tc/9372)): 

Référence papier
Pierre-Olivier Dittmar et Yann Philippe Tastevin & Annabel Vallard, « L’opium du peuple », Techniques & Culture, 70 | 2018, 6-11.
Référence électronique
Pierre-Olivier Dittmar et Yann Philippe Tastevin & Annabel Vallard, « L’opium du peuple », Techniques & Culture [En ligne], 70 | 2018, mis en ligne le 06 décembre 2018, consulté le 28 février 2019. URL : http://journals.openedition.org/tc/9372 

I have just submitted a paper to the journal, having one title from Techniques & culture in my bibliography. They admitted having internal debate on the way to write their own title (_sic_): "nous avions statué pour harmoniser dans ce sens [without space around &] depuis 2015. Mais le débat, sensible, n’est pas clos".
At this point, as you wish. I can remove the spaces.